### DISCUSSION TITLE
Welcome to BLACC Discussions 🚀

### DISCUSSION THREAD

**Discussion by blaccsmith** [blaccsmith/website#21] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, **comment below with an introduction of yourself and tell us what you're interested in 😅**.

**Comment by gt-codes** [comment]
Yeerrrrr 🗣 I'm Garrett. Any developers in Dallas 👀?
